### PR TITLE
Add lightning log message explanation doc

### PR DESCRIPTION
**Pull request by db-will**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added doc to explain lightning log message, since it binds with specific tidb version. so we add it as part of 5.4 release currently

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: NA
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+```
+[INFO] [lightning.go:233] [cfg] [cfg="{\"id\":1650510440481957437,\"lightning\":{\"table-concurrency\":6,\"index-concurrency\":2,\"region-concurrency\":8,\"io-concurrency\":5,\"check-requirements\":true,\"meta-schema-name\":\"lightning_metadata\", ...
+```
+[lightning.go:233](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/lightning.go#L233):
+Print TiDB Lightning config information.
+
+```
+[INFO] [lightning.go:312] ["load data source start"] 
+```
+[lightning.go:312](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/lightning.go#L312): Start to [scan data source dir or external storage]((https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/mydump/loader.go#L205)) defined in Lightning [mydumper data-source-dir config field](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/config/config.go#L447), and load all data source file meta info into [internal data structure](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/mydump/loader.go#L82) for future usage.
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:32]:
```suggestion
[lightning.go:312](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/lightning.go#L312): Start to [scan data source dir or external storage](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/mydump/loader.go#L205) defined in Lightning [mydumper data-source-dir config field](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/config/config.go#L447), and load all data source file meta info into [internal data structure](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/mydump/loader.go#L82) for future usage.
```

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+[INFO] [pd.go:423] ["pause configs successful at beginning"] [cfg="{\"enable-location-replacement\":\"false\",\"leader-schedule-limit\":4,\"max-merge-region-keys\":0,\"max-merge-region-size\":0,\"max-pending-peer-count\":2147483647,\"max-snapshot-count\":40,\"region-schedule-limit\":40}"]
+```
+
+[pd.go:415](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/pdutil/pd.go#L415), [pd.go:423](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/pdutil/pd.go#L423): In local backend mode, some [pd schedulers](https://docs.pingcap.com/tidb/stable/tidb-scheduling) are disabled and some [config settings are changed](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/pdutil/pd.go#L417) to split and scatter tikv regions and ingest ssts.
+
+```
+[INFO] [restore.go:1683] ["switch to import mode"]
+```
+[restore.go:1683](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/lightning/restore/restore.go#L1683):
+In local backend mode, it turns each TiKV node into import mode to speed up import process, but sacrifices its storage space. If it uses tidb backend mode, it does not need to switch TiKV to import mode.
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:106]: maybe we can link to "import mode" in https://docs.pingcap.com/tidb/stable/tidb-lightning-glossary#import-mode

**Review by lance6716 — APPROVED** [review]
And for other versions, we still can add a link to this v5.4-specific page?

**Comment by db-will** [comment]
@lance6716  thanks for reviewing. just updated

**Review by gozssky — APPROVED** [review]

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
+---
+title: TiDB Lightning Log Message Explanation
+summary: Provide detailed explanation of log messages generated during the importing process using TiDB Lightning.
+---
+
+# Introduction
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:6]:
```suggestion
# TiDB Lightning Log Message Explanation
```

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+Based on a successful test data import, this document explains log messages of **v5.4 TiDB Lightning** with **Local Backend mode**, and dive deep to understand where the log comes from and what it actually represents. Users of TiDB Lightning can refer to this doc to have a better understanding of TiDB Lightning logs. We will continue working on other TiDB versions and improving the log messages.
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:8]:
```suggestion
Based on a successful test data import, this document explains log messages of **TiDB Lightning v5.4** with **Local Backend mode**, and dives deep to understand where the log comes from and what it actually represents. You can refer to this doc to better understand TiDB Lightning logs.
```

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+
+We expect you are already familiar with TiDB Lightning, and read through the high level Lightning workflow doc in the [previous section](https://docs.pingcap.com/tidb/v5.4/tidb-lightning-overview). You can also refer to [glossary](https://docs.pingcap.com/tidb/v5.4/tidb-lightning-glossary) when you encounter an unfamiliar concept later in this doc.
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:10]:
```suggestion
To read this document, you are expected to be already familiar with TiDB Lightning and have prior knowledge of its high-level workflow described in [TiDB Lightning Overview](/tidb-lightning/tidb-lightning-overview.md). If you encounter unfamiliar terminology, you can refer to the [glossary](/tidb-lightning/tidb-lightning-glossary.md).
```

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+
+You could use this doc to quickly navigate within Lightning source code to better understand how it works internally and what exactly the log message means for.
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:12]:
```suggestion
You could use this doc to quickly navigate within TiDB Lightning source code and better understand how it works internally and what exactly the log message means.
```

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+
+Note that some trivial logs are ignored. Only important logs are included in the following doc.
+
+## Log Message Explanation
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:16]:
```suggestion
## Log message explanation 

```

**Comment by db-will** [comment]
@ran-huang could please you review the pr again?

**Comment by ran-huang** [comment]
> @ran-huang could please you review the pr again?

Sure. Thanks for the reminder!

File: tidb-lightning/tidb-lightning-log-message-explanation.md [hunk 8]
```diff
@@ -0,0 +1,360 @@
...
+
+Note that some trivial logs are ignored. Only important logs are included in the following doc.
+
+## Log message explanation 
+
+```
+[INFO] [info.go:49] ["Welcome to TiDB-Lightning"] [release-version=v5.4.0] [git-hash=55f3b24c1c9f506bd652ef1d162283541e428872] [git-branch=HEAD] [go-version=go1.16.6] [utc-build-time="2022-04-21 02:07:55"] [race-enabled=false]
+```
+
+[info.go:49](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/version/build/info.go#L49): Print TiDB Lightning version information.
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-log-message-explanation.md:22]: Can you provide permanent links here? I was told these packages would soon be moved to somewhere else. I'm concerned these links will be broken very soon. [ref](https://github.com/pingcap/tidb/blob/v5.4.0/br/pkg/version/build/info.go#L49)

**Review by ran-huang — APPROVED** [review]

**Pull request merged**